content/this-month/2022-01/index.md

```diff
@@ -82,6 +82,23 @@ Colin Finck will [talk about NTFS](https://fosdem.org/2022/schedule/event/misc_n
 The talk is on Saturday, 5 February at 17:00 (CET, UTC+1).
 The conference is virtual and admission is free.
 
+
+### [`phip1611/noto-sans-mono-bitmap-rs`](https://github.com/phip1611/noto-sans-mono-bitmap-rs)
+
+<span class="gray">(Section written by [@phip1611](https://github.com/phip1611))</span>
+
+Philipp Schuster released an initial version of his [noto-sans-mono-bitmap](https://github.com/phip1611/noto-sans-mono-bitmap-rs)
+crate this month. It is a replacement for legacy bitmap fonts, such as the [font8x8 crate](https://crates.io/crates/font8x8).
```

Comment:
I think we should first say what the crate provides, before mentioning the crates that it can replace. For example, I like the sentence "Pre-rasterized bitmap font from "Noto Sans Mono", an open font from Google." from your Readme.